spec/requests/irs_attempts_api_spec.rb

```diff
@@ -1,57 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe 'IRS attempts API' do
-  before do
-    allow(IdentityConfig.store).to receive(:irs_attempt_api_enabled).and_return(true)
-    events_to_acknowledge
-    events_to_render
-  end
-
-  let(:events_to_acknowledge) do
-    3.times.map do
-      event = IrsAttemptsApi::AttemptEvent.new(
-        event_type: :test_event,
-        session_id: 'test-session-id',
-        occurred_at: Time.zone.now,
-        event_metadata: {},
-      )
-      jti = event.jti
-      jwe = event.to_jwe
-      IrsAttemptsApi::RedisClient.new.write_event(jti: jti, jwe: jwe)
-      [jti, jwe]
-    end
-  end
-  let(:events_to_render) do
-    3.times.map do
-      event = IrsAttemptsApi::AttemptEvent.new(
-        event_type: :test_event,
-        session_id: 'test-session-id',
-        occurred_at: Time.zone.now,
-        event_metadata: {},
-      )
-      jti = event.jti
-      jwe = event.to_jwe
-      IrsAttemptsApi::RedisClient.new.write_event(jti: jti, jwe: jwe)
-      [jti, jwe]
-    end
-  end
-
-  it 'allows events to be acknowledged and renders new events' do
```

Comment:
With no event acknowledgement, it didn't seem worthwhile to rewrite this test to something else since controller tests seem to be sufficient to cover the changes here?